.eslintignore

```diff
@@ -0,0 +1,4 @@
+lib/*
```

Comment:
As they aren't essential to load-runner (either as a cli or lib), I left them out. There are >200 problems (105 errors) when including those other files.
I'm inclined to leave them out.